buildspec_clang.yml

```diff
@@ -41,6 +41,8 @@ phases:
   build:
     commands:
       - cd /build_container
+      - echo $PYTHONPATH
+      - export PYTHONPATH=/usr/local/lib
```

Comment:
This is already set when I run the docker containers manually but CodeBuild seems to run things a little differently.  Weird, since the whole idea of a container is portability.  Anyway, I'm ok with adding this if it fixes things.  But, maybe remove the preceding echo statement or move it to after it is set.

Reply:
I suspected that this could be the issue. But it looks like PYTHONPATH is set correctly as you mentioned. I cannot really reproduce these errors in containers outside CodeBuild...
I don't see xarray installed in clang and intel19 containers. Do you remember if you have installed it for these containers?